reports.py

```diff
@@ -694,19 +717,27 @@ def plot_coverage(
             colors = list(plt.rcParams['axes.prop_cycle'].by_key()['color'])    # get the colors for this style
             segment_color = colors[segment_num % len(colors)]    # pick a color, offset by the segment index
 
+            x_values = range(prior_domain_max, domain_max)
```

Comment:
Thanks for tidying up the repeated range statements.

Reply:
Welcome. :)